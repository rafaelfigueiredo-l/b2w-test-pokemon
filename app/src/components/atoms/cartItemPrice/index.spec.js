import CartItemPrice from "./";

describe("<CartItemPrice />", () => {
  const component = <CartItemPrice>Texto</CartItemPrice>;

  const mounted = mount(component);
  const wrapper = shallow(component);

  it("should mount correctly CartItemPrice component", () => {
    expect(wrapper.find(".cartItemPrice")).toHaveLength(1);
    expect(mounted).toMatchSnapshot();
  });

  it("should unmount correctly CartItemPrice component", () => {
    expect(mounted.unmount()).toEqual({});
  });

  it("should have a children", () => {
    expect(wrapper.find(".cartItemPrice").children()).toHaveLength(1);
  });
});
