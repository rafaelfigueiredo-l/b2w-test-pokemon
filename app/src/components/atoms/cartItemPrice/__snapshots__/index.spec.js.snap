// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CartItemPrice /> should mount correctly CartItemPrice component 1`] = `
<styled.span
  className="cartItemPrice"
>
  <span
    className="sc-AxjAm StDqN cartItemPrice"
  >
    Texto
  </span>
</styled.span>
`;
